packages/styled-ui-docs/pages/toast.mdx

```diff
@@ -6,7 +6,55 @@ A toast notification is a short message that displays at the  side of the screen
 
 A `Toast` element works with the `useToast` hook to trigger a toast notification.
```

Comment:
Can be modified to 

A Toast element can be used with the useToast hook to trigger a Toast notification.